SettingsList.py

```diff
@@ -2884,10 +2884,10 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'any_dungeon':      {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
             'keysanity':        {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
             'lacs_vanilla':     {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
-            'lacs_stones':      {'settings': ['lacs_medallions', 'lacs_rewards', 'lacs_tokens']},
-            'lacs_medallions':  {'settings': ['lacs_stones', 'lacs_rewards', 'lacs_tokens']},
-            'lacs_dungeons':    {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_tokens']},
-            'lacs_tokens':      {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards']},
+            'lacs_stones':      {'settings': ['triforce_hunt', 'lacs_medallions', 'lacs_rewards', 'lacs_tokens']},
+            'lacs_medallions':  {'settings': ['triforce_hunt', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
+            'lacs_dungeons':    {'settings': ['triforce_hunt', 'lacs_medallions', 'lacs_stones', 'lacs_tokens']},
+            'lacs_tokens':      {'settings': ['triforce_hunt', 'lacs_medallions', 'lacs_stones', 'lacs_rewards']},
```

Comment:
It's a bit hard to explain so here is what I meant.
I haven't tested it but if disables function as I think they do, it should work.
```suggestion
            'lacs_stones':      {'settings': ['lacs_medallions', 'lacs_rewards', 'lacs_tokens']},
            'lacs_medallions':  {'settings': ['lacs_stones', 'lacs_rewards', 'lacs_tokens']},
            'lacs_dungeons':    {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_tokens']},
            'lacs_tokens':      {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards']},
            'triforce':         {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
```